pkg/placementrequestbinderargs/v1alpha1/schedulerconfig.go

```diff
@@ -0,0 +1,18 @@
+package v1alpha1
```

Comment:
`pkg/apis/scheduler/v1alpha1` is a better place for any sort of defaulting of the arguments.